src/index.js

```diff
@@ -594,10 +593,12 @@ function cov2ap(options = {}) {
           // Trace
           traceRequest(req, "ProxyRequest", "POST", postUrl, postHeaders, body);
 
+          const postBody = JSON.stringify(body);
+          postHeaders["content-length"] = postBody.length;
```

Comment:
Isn't fetch Node 18 or later only?

At least that is stated in https://developer.mozilla.org/en-US/docs/Web/API/fetch#browser_compatibility

And your package.json says that Node 16 or later is supported: https://github.com/cap-js-community/odata-v2-adapter/blob/ae1f7640a8f921c2880524d5d894ec5172b9dd5d/package.json#L7

Reply:
fetch is released for Node 18, but available since Node 16 (but you have to set a experimental flag). 
I think, I will drop Node 16 support. Thanks for the hint.